src/test/common/steps/privacy-notice-steps.js

```diff
@@ -0,0 +1,38 @@
+const { When, Then } = require('cucumber')
+const { expect } = require('chai')
+
+const pages = require('./pages')
+
+const GOV_UK_TABLE_CLASSNAME = 'govuk-table'
+
+When(/^I click on the privacy-notice link$/, async function () {
+  await pages.genericPage.clickPrivacyNoticeLink()
+})
+
+Then(/^the privacy-notice page is shown$/, async function () {
+  await pages.privacyNotice.waitForPageLoad()
+})
+
+Then(/^all page content is present on the privacy-notice page$/, async function () {
+  await checkPrivacyNoticePageContentIsPresentAndCorrect()
+})
+
+Then(/^the back link on the privacy-notice page links to the enter name page$/, async function () {
+  const backLink = await pages.privacyNotice.getBackLink()
+  const href = await backLink.getAttribute('href')
+
+  // using startsWith() instead of equals() so query parameters do not fail the test
+  const enterNameUrl = `${pages.url}${pages.enterName.getPath()}`
+  expect(href.startsWith(enterNameUrl)).to.equal(true)
```

Comment:
You can use Chai's `startWith`, so something like: `expect('abcdef').to.startsWith('abc')` or `'abcdef'.should.startWith('abc')` if the grammar of the first doesn't seem quite right.

Reply:
Gethyn thinks it requires bringing in a new package - not worthwhile for this one assertion

Reply:
That's fine.